config/tilokit.yaml

```
@@ -37,6 +37,20 @@ frameworks:
     typescript: true
     testing: "vitest"
     
+  next:
+    default_build_tool: "next"
+    typescript: true
+    testing: "jest"
+    linting: true
+    formatting: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)